tests/e2e/helpers/database-scripts.ts

```diff
@@ -3,70 +3,111 @@ import { workingDirectory } from '../helpers/conf';
- * @param tableName The name of table in DB
- */
-export async function deleteRowsFromTableInDB(tableName: string): Promise<void> {
-    const db = new sqlite3.Database(dbPath);
-    const query = `DELETE FROM ${tableName}`;
+    /**
+     * Get Column value from table in local Database
+     * @param dbTableParameters The sqlite database table parameters
+     */
+    static async getColumnValueFromTableInDB(dbTableParameters: DbTableParameters): Promise<any> {
+        const rowIndex = await DatabaseScripts.getRowIndexFromTableInDB(dbTableParameters);
+        if (rowIndex !== null) {
+            const db = new sqlite3.Database(dbPath);
+            const query = `SELECT ${dbTableParameters.columnName} FROM ${dbTableParameters.tableName} WHERE rowid = ?`;
+
+            return new Promise<void>((resolve, reject) => {
+                db.get(query, [rowIndex], (err: { message: string }, row: any) => {
+                    if (err) {
+                        reject(new Error(`Error during getting ${dbTableParameters.columnName} column value: ${err.message}`));
+                    }
+                    else {
+                        const columnValue = row[dbTableParameters.columnName!];
+                        db.close();
+                        resolve(columnValue);
+                    }
+                });
+            });
+        }
+        throw new Error(`No matching row found for the given condition in ${dbTableParameters.tableName}`);
+
+    }
 
-    return new Promise<void>((resolve, reject) => {
+    /**
+     * Delete all rows from table in local DB
+     * @param dbTableParameters The sqlite database table parameters
+     */
+    static async deleteRowsFromTableInDB(dbTableParameters: DbTableParameters): Promise<void> {
+        const db = new sqlite3.Database(dbPath);
+        const query = `DELETE FROM ${dbTableParameters.tableName}`;
 
-        db.run(query, (err: { message: string }) => {
-            if (err) {
-                reject(new Error(`Error during ${tableName} table rows deletion: ${err.message}`));
-            }
-            else {
-                db.close();
-                resolve();
-            }
+        return new Promise<void>((resolve, reject) => {
+
+            db.run(query, (err: { message: string }) => {
+                if (err) {
+                    reject(new Error(`Error during ${dbTableParameters.tableName} table rows deletion: ${err.message}`));
+                }
+                else {
+                    db.close();
+                    resolve();
+                }
+            });
         });
-    });
+    }
 }
+/**
+ * Add new database parameters
+ * @param tableName The name of table in DB
+     * @param columnName The name of column in table
+     * @param rowValue Value to update in table
+     * @param conditionColumnName The name of the column to search
+     * @param conditionColumnValue The value to match in the column
+ */
+export type DbTableParameters = {
```

Comment:
what is the different between conditionColumnName and columnName?
and pleases clarify me, how exactly we should specify what row should be undated?

Reply:
renamed to conditionWhereColumnName and conditionWhereColumnValue